README.md

```diff
@@ -68,6 +68,37 @@ Token is stored inside the instance and inside browser `localStorage`.
 
 Emits `login` and `tokenChange` events.
 
+##### `.loginWithToken(providerName: String, loginToken: String)`
+
+Attempt to login to Reindex with a login token provided by some provider.
+Currently only Auth0 (`auth0`) is supported. Reindex validates the token on
+the backend side and performs normal login there. As no browser interaction is
+required from user, it can be used on server or in React Native.
+Otherwise behaves as `login`: sets token and stores token in `localStorage`
+(if available).
+
+Emits `login` and `tokenChange` events.
+
+Example usage with Auth0 lock:
+
+```js
+const lock = new Auth0Lock('YOUR-AUTH0-CLIENT-ID', 'YOUR-AUTH0-DOMAIN');
+lock.show({}, async (err, auth0Token) => {
+  if (err) {
+    console.log(err);
+    return;
+  }
+
+  const reindexResponse = await reindex.loginWithToken(
+    'auth0',
+    auth0Result[1],
+  );
+  for (const error of reindexResponse.errors || []) {
+    console.log(error);
+  }
```

Comment:
Please copy the fixed example code from docs: https://www.reindex.io/docs/security/authentication/#login-with-auth0-lock